auth-server/src/model/PasswordResetToken.ts

```diff
@@ -0,0 +1,47 @@
+import * as Knex from 'knex'
+import { Model } from 'objection'
+import * as connection from '../../knexfile'
+
+const knexConnection = Knex(connection)
+
+Model.knex(knexConnection)
+
+export default class PasswordResetToken extends Model {
+	readonly id!: number
+	token!: string
+	user_id!: number
+	valid: boolean
+	expires: number
```

Comment:
type `Date` should work. (please also change it for `RefreshToken`)

Reply:
iso string should be used. What do you think

Reply:
Sure, sounds good.